### PR TITLE
RenaultBank Importer Fixes

### PR DESCRIPTION
**Pull request by ZfT2**
Some more fixes for newer Account Statements and a little bit refactoring :)

### CODE REVIEW

**Comment by auchri** [comment]
Should this importer be also split into two @Nirus2000 @buchen ?

It contains Renault Bank DE and Renault Bank direkt AT

File: name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/RenaultBankDirektPDFExtractor.java [hunk 1]
```diff
@@ -16,18 +13,19 @@
 @SuppressWarnings("nls")
 public class RenaultBankDirektPDFExtractor extends AbstractPDFExtractor
 {
-    private static final String DEPOSIT_2019 = "^(?<date>\\d+.\\d+.)  (.*-Gutschrift)(\\s*)(?<amount>[\\d\\s,.]*)(\\+$)";
-    private static final String REMOVAL_2019 = "^(?<date>\\d+.\\d+.)  (Internet-Euro-Überweisung)(\\s*)(?<amount>[\\d\\s,.]*)(\\-)(.*)";
-    private static final String INTEREST_2019 = "^(?<date>\\d+.\\d+.)  (.*)(Zinsen\\/Kontoführung)(\\s*)(?<amount>[\\d\\s,.]*)(\\+$)";
+    private static final String DEPOSIT_2019 = "^(?<date>\\d+.\\d+.)  (.*-Gutschrift)(\\s*)(?<amount>[\\d\\s,.]*)[+](\\s*)$";
+    private static final String REMOVAL_2019 = "^(?<date>\\d+.\\d+.)  (Wertstellung: \\d+.\\d+. )?(Internet-Euro-Überweisung)(\\s*)(?<amount>[\\d\\s,.]*)[-](\\s*)$";
```
Comment (Nirus2000) [inline name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/RenaultBankDirektPDFExtractor.java:17]: the same like the https://github.com/portfolio-performance/portfolio/pull/3775... with comment https://github.com/portfolio-performance/portfolio/pull/3775#discussion_r1481827304

The rest look quite good!! 💯

**Review by Nirus2000 — COMMENTED** [review]
Hey @ZfT2
check my comments... i think it looks good with a few easy modifications... 👍🏻

Good Job...

File: name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/RenaultBankDirektPDFExtractor.java (hunk 1 above)
Comment (ZfT2) [reply name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/RenaultBankDirektPDFExtractor.java]: ok, changed.

**Comment by ZfT2** [comment]
Thanks again... Changed it :)

**Pull request merged**